tests/run-make/x86_64-fortanix-unknown-sgx-lvi/rmake.rs

```diff
@@ -0,0 +1,92 @@
+    check("__libunwind_Registers_x86_64_jumpto", "jumpto.checks");
+
+    check("std::io::stdio::_print::[[:alnum:]]+", "print.with_frame_pointers.checks");
+
+    check("st_plus_one_global_asm", "rust_plus_one_global_asm.checks");
+
+    check("_plus_one_c", "cc_plus_one_c.checks");
+    check("_plus_one_c_asm", "cc_plus_one_c_asm.checks");
+    check("_plus_one_cxx", "cc_plus_one_cxx.checks");
+    check("_plus_one_cxx_asm", "cc_plus_one_cxx_asm.checks");
+    check("_plus_one_asm", "cc_plus_one_asm.checks");
+
+    check("cmake_plus_one_c", "cmake_plus_one_c.checks");
+    check("cmake_plus_one_c_asm", "cmake_plus_one_c_asm.checks");
+    check("cmake_plus_one_c_global_asm", "cmake_plus_one_c_global_asm.checks");
+    check("cmake_plus_one_cxx", "cmake_plus_one_cxx.checks");
+    check("cmake_plus_one_cxx_asm", "cmake_plus_one_cxx_asm.checks");
+    check("cmake_plus_one_cxx_global_asm", "cmake_plus_one_cxx_global_asm.checks");
+    check("cmake_plus_one_asm", "cmake_plus_one_asm.checks");
+}
+
+fn check(func_re: &str, checks: &str) {
+    let dump = llvm_objdump()
+        .input("enclave/target/x86_64-fortanix-unknown-sgx/debug/enclave")
+        .args(&["--syms", "--demangle"])
+        .run()
+        .stdout_utf8();
+    let re = regex::Regex::new(&format!("[[:blank:]]+{func_re}")).unwrap();
+    let func = re.find_iter(&dump).map(|m| m.as_str().trim()).collect::<Vec<&str>>().join(",");
+    let dump = llvm_objdump()
+        .input("enclave/target/x86_64-fortanix-unknown-sgx/debug/enclave")
+        .args(&["--syms", &format!("--disassemble-symbols={func}")])
+        .run()
+        .stdout_utf8();
+    let dump = dump.as_bytes();
+
+    // Unique case, must succeed at one of two possible tests.
+    if func_re == "std::io::stdio::_print::[[:alnum:]]+" {
+        let output = llvm_filecheck().stdin(&dump).patterns(checks).run_unchecked();
+        if !output.status().success() {
+            llvm_filecheck().stdin(&dump).patterns("print.without_frame_pointers.checks").run();
+            llvm_filecheck()
+                .args(&["--implicit-check-not", "ret"])
+                .stdin(dump)
+                .patterns("print.without_frame_pointers.checks")
+                .run();
+        } else {
+            llvm_filecheck()
+                .args(&["--implicit-check-not", "ret"])
+                .stdin(dump)
+                .patterns(checks)
+                .run();
+        }
+        return;
+    }
+    llvm_filecheck().stdin(&dump).patterns(checks).run();
+    if !["rust_plus_one_global_asm", "cmake_plus_one_c_global_asm", "cmake_plus_one_cxx_global_asm"]
+        .contains(&func_re)
+    {
+        // The assembler cannot avoid explicit `ret` instructions. Sequences
+        // of `shlq $0x0, (%rsp); lfence; retq` are used instead.
+        // https://www.intel.com/content/www/us/en/developer/articles/technical/
+        //     software-security-guidance/technical-documentation/load-value-injection.html
```

Comment:
Suggestion: this link should not be wrapped, please add a `ignore-tidy-linelength` if you have to.